tensorboard/plugins/histogram/tf_histogram_dashboard/tf-histogram-loader.html

```diff
@@ -111,11 +163,15 @@
         },
         getDataLoadUrl: {
           type: Function,
-          value: () => ({tag, run}) => {
+          value: () => ({tag, run, experiment}) => {
```

Comment:
Again, the experiment change doesn't seem necessary since neither _csvUrl or _jsonUrl below actually use this parameter.  Can we leave it unchanged unless there's a reason you need it?